http/oidc/src/main/java/org/wildfly/security/http/oidc/BasicAuthRequestAuthenticator.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2022 Red Hat, Inc., and individual contributors
```

Comment:
2023?

Reply:
All of the commits in this PR except the last one are just backports of existing commits from the 1.x branch.